_episodes/46.md

```diff
@@ -0,0 +1,152 @@
+be familiar with like SQL Server, Redis, and SingleStore. If you're not familiar
+with them or with in-memory computing, the gist is that by focusing on using RAM
+instead of disk storage, you can create a database system which is *much*
+faster - the Ignite website advertises 10-1000x improvements. Of course, this is
+more expensive, too, so it thrives in settings where performance is critical. 
+
+With an initial release 7 years ago, Ignite is still something of the new kid on
+the block among in-memory databases, coming with modern bells and whistles that
+has it positioned to become a successor to those other, comparable databases
+mentioned above. It also has some key functionality that sets it apart.
+
+## Ignite + Trino
+
+
+
+## Contributing the Ignite connector
+
+The Trino community and developers try their best to be active reviewers,
+collaborators, and participants on pull requests coming in from outside
+contributors. Massive contributions like the Ignite connector can take a lot of
+round trips, back-and-forth discussion, and work from both the contributor and
+the project's maintainers to get it into a state where it is ready to merge and
+go live for users to try out.
+
+To give you an idea, the pull request (PR) to contribute Ignite was opened in
+mid-June, 2021. It received immediate feedback from a couple maintainers, went
+through a few round trips with amendments, re-reviews, more edits, and then
+other reviews. But in an open source environment, each round trip can tend to
+take longer and longer. Progress stalled in November 2021, and both Jason and
+the maintainers didn't poke the Ignite PR for nearly a year. In October 2022, as
+part of Trino DevRel's roundup of stale and out-of-date pull requests, we bumped
+back into the work that Jason had done. The wheels began to turn again, starting
+slow but picking up the pace, until it returned to full and active development,
+with several maintainers checking in frequently until the connector was ready to
+go. But that's the story from an observer, and we've got Jason here to go into
+more detail.
+
+### Questions for Jason
+
+* How was the Trino review process?
+* Were there any major lessons you picked up along the way?
+* What tips would you give to someone else looking to add something into Trino?
+
+## PR of the episode: #13943: Add support for `migrate` procedure in Iceberg
+
+If you've been in the data space for a while, you may know that there's a bit of
+a prevailing current in migrating from Hive to Iceberg. Out with the old, in
+with the new, and in with the performance gains. [Yuya Ebihara](https://github.com/ebyhr),
+one of the Trino maintainers, has added a table procedure to Trino's Iceberg
+connector to make that process much, much simpler. Rather than a slow, manual,
+and arduous process, if you have a Hive table stored in a file format supported
+by Iceberg, it's as simple as calling the `migrate` table procedure and letting
+it run. Neat, right?
```

Comment:
Do we want to talk about what this does? Even high level  .. I assume it creates metadata in the Iceberg format, wipes most metadata in the HMS, and then updates the HMS to reference the iceberg format